mesh/mesh.hpp

```diff
@@ -1558,6 +1728,17 @@ class Mesh
       operator Mesh::FaceInfo() const;
    };
 
+   /// @name More advanced entity information access methods
+   /// @{
+
+   /* Return point matrix of element i of dimension Dim X #v, where for every
```

Comment:
Could probably use an explanation of what the point matrix is here.  Which points.  Are they in the columns or rows of the matrix?